### PR TITLE
Update LICENSE and contributing guideline

### PR DESCRIPTION
**Pull request by daico007**
Update LICENSE format and add contribution guideline, as suggested in #101 and #97.

### CODE REVIEW

**Comment by bc118** [comment]
**This also needs to be added to the docs, when you get a chance.  I am thinking we can put it in the docs under:**
- Getting Started
- References
- or create a new Contributing section

**Comment by daico007** [comment]
Added to docs under `References`

File: docs/reference/contributing.rst [hunk 1]
```diff
@@ -0,0 +1,17 @@
+================================
+How to Contribute to the Project
+================================
+
```
Comment (bc118) [inline docs/reference/contributing.rst:4]: My only comment is to make this the same as the "CONTRIBUTING.md" file above. The CONTRIBUTING.md file has added text in it.

**Review by bc118 — CHANGES_REQUESTED** [review]

File: docs/reference/contributing.rst (hunk 1 above)
Comment (daico007) [reply docs/reference/contributing.rst:4]: I updated the contributing in docs to match with the `CONTRIBUTING.md`

**Review by bc118 — APPROVED** [review]

**Pull request merged**